### PR TITLE
Print out hotel names

### PR DESCRIPTION
**Pull request by mikeymurph77**
@jsteiner, @gabebw

### CODE REVIEW

File: hotlr/hotel_reader.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class HotelReader
+  def initialize(filename)
+    @filename = filename
+  end
+  
+  def read
+    hotel = {}
+    
+    CSV.foreach(@filename, :headers => true) do |row|
```
Comment (gabebw) [inline hotlr/hotel_reader.rb]: Use fancy hashes: `headers: true`

File: hotlr/hotel_reader.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require "csv"
+require "pp"
+
+class HotelReader
+  def initialize(filename)
+    @filename = filename
+  end
+  
+  def read
+    hotel = {}
```
Comment (gabebw) [inline hotlr/hotel_reader.rb]: Do you use this variable?

File: hotlr/hotel_reader.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    
+    CSV.foreach(@filename, :headers => true) do |row|
+      puts row[0]
+    end
+  end
+
```
Comment (gabebw) [inline hotlr/hotel_reader.rb]: extra line, remove

**Comment by mikeymurph77** [comment]
@jsteiner, @gabebw 
I fixed my CSV block so I can call the headers by their name and not by their array value.
Am I ready to move on?

**Comment by gabebw** [comment]
:+1: Good to merge!

**Pull request merged**